lms/djangoapps/verified_track_content/tests/test_views.py

```diff
@@ -0,0 +1,73 @@
+"""
+Tests for verified track content views.
+"""
+
+import json
+
+from nose.plugins.attrib import attr
+from unittest import skipUnless
+
+from django.http import Http404
+from django.test.client import RequestFactory
+from django.conf import settings
+
+from student.tests.factories import UserFactory, AdminFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from verified_track_content.models import VerifiedTrackCohortedCourse
+from verified_track_content.views import cohorting_settings
+
+
+@attr('shard_2')
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Tests only valid in LMS')
+class CohortingSettingsTestCase(SharedModuleStoreTestCase):
+    """
+    Tests the `cohort_discussion_topics` view.
+    """
+    SAMPLE_COURSE = 'edX/Test_Course/Run'
```

Comment:
This doesn't seem to be used anywhere, and it would make more sense to use `self.course.id` anyway (which you already seem to be doing).